### ISSUE TITLE
Error in Package Installation and Documentation Generation

### ISSUE THREAD

**Issue by Soumya624** [GeomScale/Rvolesti#13]
When trying to install the package or create it's documentation,  a library specific error is generating. To replicate this within RStudio, one can execute the following two commands:

```
Rcpp::compileAttributes()
roxygen2::roxygenize(roclets="rd")
```

**Comment by vissarion** [comment]
Is this still open or fixed? Could you please provide more details on the error?

**Comment by Soumya624** [comment]
> Is this still open or fixed? Could you please provide more details on the error?

The error is still coming while we're trying to build the documentation. For me, it's showing the following message:

![image](https://github.com/GeomScale/Rvolesti/assets/63846042/1b3fa3e2-8e7a-4b59-9d21-c9a35e0952e6)

**Comment by vfisikop** [comment]
This is probably fixed by https://github.com/GeomScale/Rvolesti/pull/15 Please check and if this is the case please close the issue.

**Comment by Soumya624** [comment]
> This is probably fixed by #15 Please check and if this is the case please close the issue.

Yes. Thanks!
